addons/website_event/views/event_templates_list.xml

```diff
@@ -204,7 +205,42 @@
                     </t>
                 </t>
             </div>
-        </li>
+        </div>
+    </xpath>
+    <xpath expr="//span[hasclass('o_wevent_offcanvas_location')]" position="replace">
+        <div class="accordion-item">
+            <h2 class="accordion-header">
+                <button class="accordion-button border-top collapsed"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    data-bs-target=".o_wevent_offcanvas_country"
```

Comment:
Same here then, of course (alternatively could change `o_wevent_offcanvas_location` to `o_wevent_offcanvas_country`)